PRIVACY_POLICY.md

```diff
@@ -1 +1 @@
-The tool only gathers information's about installed and missing updates, all data are processed local on the users own device, no personal information's of any kind are send to the cloud.
\ No newline at end of file
+The tool only gather information about installed and missing updates, all data are processed locally on the user's own device, no personal information of any kind is sent to the cloud.
```

Comment:
* `gather` should be `gathers`
* `all data are processed` should be `all data is processed` (grammar)